### PR TITLE
20 add create message post

### LINKED ISSUE #20: Add create_message POST

### LINKED ISSUE #65: refactor: move axios into devDependencies

### PR DESCRIPTION
**Pull request by matthewsong18**
I had to change my chat_controller to message_controller because I'm not going to deal with fixing all the overlapping function names. Idk why but somehow chatController has methods with exact copies of my method names. Like respond_with_error_json and such. Decided not to deal with that and just rename.

### CODE REVIEW

File: backend/controllers/message_controller.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
+const Chat = require("../models/Chat.js");
+const ChatService = require("../services/ChatService.js");
+const User = require("../models/User.js");
```
Comment (BaoNguyen09) [inline backend/controllers/message_controller.js:3]: I think you should use UserService here like we agreed on

Comment (matthewsong18) [reply backend/controllers/message_controller.js:3]: Why? I'm not creating any user?

**Review by BaoNguyen09 — DISMISSED** [review]
Looks good to me, not much code conflicts with mine branch so it should be fine to merge

**Review by BaoNguyen09 — APPROVED** [review]
Looks good.

**Pull request merged**